wallet/internal/db/block_sqlite.go

```diff
@@ -0,0 +1,32 @@
+package db
+
+import (
+	"database/sql"
+	"fmt"
+	"time"
+
+	"github.com/btcsuite/btcd/chaincfg/chainhash"
+)
+
+// buildSqliteBlock constructs a Block from the given SQLite block
+// fields.
+func buildSqliteBlock(
+	height sql.NullInt64, hash []byte, timestamp sql.NullInt64,
```

Comment:
looks like the only diff from `buildPgBlock` is the `height` type? I think we can just use sth like [this](https://github.com/lightningnetwork/lnd/blob/20473482d45f6cb5ef45ee4b80b6505555d9cc1f/tlv/primitive.go#L379).